tests/v2/testlib/waku2.nim

```diff
@@ -0,0 +1,58 @@
+import
+  std/options,
+  stew/byteutils,
+  libp2p/switch,
+  libp2p/builders
+import
+  ../../../waku/v2/protocol/waku_message,
+  ./common
+
+export switch
+
+
+# Switch
+
+proc generateEcdsaKey*(): PrivateKey =
+  PrivateKey.random(ECDSA, rng[]).get()
+
+proc generateEcdsaKeyPair*(): KeyPair =
+  KeyPair.random(ECDSA, rng[]).get()
+
+proc generateSecp256k1Key*(): PrivateKey =
+  PrivateKey.random(Secp256k1, rng[]).get()
+
+proc generateKey*(): PrivateKey =
+  generateSecp256k1Key()
+
+proc newTestSwitch*(key=none(PrivateKey), address=none(MultiAddress)): Switch =
+  let peerKey = key.get(generateKey())
+  let peerAddr = address.get(MultiAddress.init("/ip4/127.0.0.1/tcp/0").get())
+  return newStandardSwitch(some(peerKey), addrs=peerAddr)
+
+
+# Waku message
+
+export
+  waku_message.DefaultPubsubTopic,
+  waku_message.DefaultContentTopic
+
```

Comment:
Unsure what's the policy on exports in the middle of the file, what are your thoughts?